lib/commands/backup.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+const Command = require('../command');
+const Promise = require('bluebird');
+const createDebug = require('debug');
+const Zip = require('adm-zip');
+const fs = require('fs');
+const path = require('path');
+const cli = require('../');
+
+const debug = createDebug('ghost-cli:backup');
+
+class BackupCommand extends Command {
+    run(argv) {
+        const instance = this.system.getInstance();
+        const config = instance.config.values;
+        const datetime = (new Date()).toJSON().substring(0, 10);
+
+        let exporter;
+        let saveLocation = process.cwd();
+
+        // @todo check if we need to ensure we're in a ghost directory
+
+        if (argv.output) { // @todo add o as an alias
+            saveLocation = path.isAbsolute(argv.output) ? argv.output : path.join(process.cwd(), argv.output);
+            fs.ensureDirSync(saveLocation);
+        }
+
+        saveLocation = path.join(saveLocation, `ghost.backup.${datetime}.zip`)
+
+        instance.checkEnvironment();
+
+        if (instance.running()) {
+            this.ui.log('Ghost is currently running. Backing up might take longer and slow down your site','yellow');
+        }
+
+        const backupData = {
+            versions: {
+                cli: this.system.cliVersion,
+                ghost: instance.cliConfig.get('active-version')
+            },
+            database: config.database.client,
+            contentPath: config.paths.contentPath,
+            // @todo: check if this is necessary
+            environment: instance.config.environment
+        }
+
+        try {
+            // @todo: check if we need to `path.join`
+            exporter = require(path.join(process.cwd(),'/current/core/server/data/export'));
+        } catch (e) {
+            console.log(e);
+            return Promise.reject(new Error('Unable to initialize database exporter'));
+        }
+
+        // @todo: Log this?
+        debug('Creating database backup');
+
+        const props = {
+            database: exporter.doExport({}),
+            environment: this.system.environment
```

Comment:
not sure why this is in the props - it's not a promise and you've got access to the environment inside the `then` callback regardless